Bubble/app/components/MyChatListComponent.js

```diff
@@ -115,7 +120,7 @@ const mapStateToProps = (state, ownProps) => {
         socket: state.socket,
         bubbleId: state.bubbleId,
         rooms: Object.values(state.rooms.data),
-        myRooms: state.myRooms.data,
+        myRooms: state.joinedRooms,
```

Comment:
this is not correct, myRooms is used in render to be sorting, joinedRooms is an array of roomIds.